README.md

```diff
@@ -150,3 +150,99 @@ Currently in Doppler [we are always using the v1](https://github.com/MakingSense
   </body>
 </html>
 ```
+## File storage
```

Comment:
Please, add a line before the title, in general in markdown the paragraphs have a line between them.

```suggestion

## File storage
```

Reply:
Changed.